### PR TITLE
Add coffeelint support

### PR DESCRIPTION
**Pull request by mnquintana**
Refs atom/atom#6230, atom/atom#5982

### CODE REVIEW

File: lib/project/results-model.coffee [hunk 1]
```diff
@@ -67,9 +67,6 @@ class ResultsModel
   onDidRemoveResult: (callback) ->
     @emitter.on 'did-remove-result', callback
 
-  onDidFinishSearching: (callback) ->
```
Comment (kevinsawicki) [inline lib/project/results-model.coffee]: Hmm, any reason to remove this? Was it just unused?

Comment (mnquintana) [inline lib/project/results-model.coffee]: It's a duplicate - it's already defined at https://github.com/mnquintana/find-and-replace/blob/coffeelint/lib/project/results-model.coffee#L46-L47

Comment (kevinsawicki) [inline lib/project/results-model.coffee]: Whoa, nice to have this catch that :+1:

**Comment by kevinsawicki** [comment]
Thanks!

**Pull request merged**